src/components/Map.tsx

```
@@ -0,0 +1,37 @@
+"use client"
+
+import React from 'react';
+import { APIProvider, Map as GoogleMap, Marker } from '@vis.gl/react-google-maps';
+
+interface MapProps {
+ center?: { lat: number; lng: number };
+ markers?: { position: { lat: number; lng: number }; title: string }[];
+}
+
+const Map = ({ center, markers }: MapProps) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)